raiden/network/protocol.py

```diff
@@ -661,36 +662,40 @@ def send_async(self, receiver_address, message):
         return async_result
 
     def send_and_wait(self, receiver_address, message, timeout=None):
-        """Sends a message and wait for the response ack."""
+        """Sends a message and wait for the response 'Processed' message."""
         async_result = self.send_async(receiver_address, message)
         return async_result.wait(timeout=timeout)
 
-    def maybe_send_ack(self, receiver_address, ack_message):
-        """ Send ack_message to receiver_address if the transport is running. """
+    def maybe_send_processed(self, receiver_address, processed_message):
+        """ Send processed_message to receiver_address if the transport is running. """
         if not isaddress(receiver_address):
             raise InvalidAddress('Invalid address {}'.format(pex(receiver_address)))
 
-        if not isinstance(ack_message, Ack):
-            raise ValueError('Use maybe_send_ack only for Ack messages')
+        if not isinstance(processed_message, Processed):
+            raise ValueError('Use _maybe_send_processed only for `Processed` messages')
 
-        messagedata = ack_message.encode()
-        self.receivedhashes_to_acks[ack_message.echo] = (receiver_address, messagedata)
-        self._maybe_send_ack(*self.receivedhashes_to_acks[ack_message.echo])
+        messagedata = processed_message.encode()
 
-    def _maybe_send_ack(self, receiver_address, messagedata):
-        """ ACK must not go into the queue, otherwise nodes will deadlock
+        self.receivedhashes_to_processedmessages[processed_message.echo] = (
+            receiver_address, messagedata)
```

Comment:
style note here. Change this to:

```python
self.receivedhashes_to_processedmessages[processed_message.echo] = (
            receiver_address,
            messagedata,
)
```

Same with `self._maybe_send_processed` below

Reply:
Made the fixes